docs/examples/milestones/implementation-milestones-3.json

```diff
@@ -0,0 +1,212 @@
+{
```

Comment:
Suggest making this a minimal release which only includes the `contract/implementation/milestones` section and required fields. Please make the other changes noted in `implementation-milestones-1.json`.